apim/3.x/README.adoc

```diff
@@ -82,13 +101,30 @@ chart)
 |`+ldap.enabled+` |Enable LDAP login |false
 |===
 
-==== Mongo
 
-===== MongoDB connections
+
+=== 3. Configure  Optional Dependencies 
+
+Specify each parameter using the `+--set key=value[,key=value]+`
```

Comment:
Suggestion, maybe those lines from 108-120 are like a "helm install command tip", and should be located in a dedicated section ?
It may be confusing cause it seems user has to launch this `$ helm install --name my-release -f values.yaml gravitee` command here. But it's just here as an example ?